docs/intro.rst

```diff
@@ -33,6 +36,9 @@ MusicBrainz account). In the ``Callback URL`` field type::
 
    ``<HOST>`` field should be set to ``localhost`` if you plan to run a local instance of
    CritiqueBrainz for development purposes.
+   For eg:- If you are running your local instance of the server on Port No.
```

Comment:
Please use `For example` instead of `for eg`. And port number instead of port no.

I prefer full and correct words over shortened versions in documentation.